lib/lev/status.rb

```diff
@@ -96,13 +145,13 @@ def has_reserved_keys?(hash)
     end
 
     def push(key, new_item)
-      new_value = (get(key) || []).push(new_item)
+      new_value = (send(key) || []).push(new_item)
       set(key => new_value)
     end
 
     STATES.each do |state|
       define_method("#{state}?") do
-        get('state') == state
+        status == state
```

Comment:
I think `status` is the more natural word as I keep referencing it as that in tests and in the tutor server controllers that will use it. I think Lev::Status might be more like Lev::BackgroundJob now, too. We only used `state` because `status.status` is weird, but if it's a `job` then `job.id` `job.progress` and `job.status` make nice sense. status.id, status.progress.. ehh